configs/MapzenAndroid.xml

```diff
@@ -1,4 +1,4 @@
-<code_scheme name="MapzenAndroid">
+<code_scheme name="SquareAndroid">
```

Comment:
Fixed here https://github.com/mapzen/java-code-styles/pull/1/commits/bdfcc3ef6bd5dfcfd059cb50ad90d1c055925202. I think you might just need to refresh.